libraries/lib-module-manager/PluginIPCUtils.h

```diff
@@ -0,0 +1,84 @@
+/**********************************************************************
+
+  Audacity: A Digital Audio Editor
+
+  @file PluginIPCUtils.h
+
+  @author Vitaly Sverchinsky
+
+  @brief Provides a set of internal commonly used functions/structures
+
+  Part of lib-module-manager library.
+
+**********************************************************************/
+
+#pragma once
+
+#include <list>
+#include <wx/string.h>
+#include "XMLTagHandler.h"
+
+class IPCChannel;
+class PluginDescriptor;
+class XMLWriter;
+
+namespace details
```

Comment:
Is the convention `details` or `detail`?  I used `detail`

Reply:
Depends on project I believe, do we have such convention?